src/header/header.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 let links = ['BLOG', 'ABOUT', 'LINKS', 'PROJECTS'];
 let linksList = links.map(function (link) {
```

Comment:
```suggestion
const links = ['BLOG', 'ABOUT', 'LINKS', 'PROJECTS'];
const linksList = links.map(function (link) {
```
try to use `let` as little as possible. `const` ensures that your variable is [immutable](https://stackoverflow.com/a/20505813/3086270) which makes your programms easier to follow